server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/SqlCheckService.java

```diff
@@ -94,7 +94,7 @@ public List<CheckResult> check(@NotNull ConnectionSession session,
     }
 
     public List<CheckViolation> check(@NotNull Long environmentId, @NonNull String databaseName,
-            @NotNull List<OffsetString> sqls, @NotNull ConnectionConfig config) {
+            @NotNull List<OffsetString> sqls, @NotNull ConnectionConfig config, List<SqlCheckRule> defaultRules) {
```

Comment:
How to understand the `defaultRules`? Additional rules? Why not create a new method such as `public List<CheckViolation> check(@NotNull Long environmentId, @NonNull String databaseName, @NotNull List<OffsetString> sqls, @NotNull ConnectionConfig config, List<SqlCheckRule> rules)`. In this method, you can specify what rules you want to use.

Reply:
you are right! Done.